src/components/label-pane.jsx

```diff
@@ -60,7 +78,7 @@ LabelPane.propTypes = {
     showUnselected: PropTypes.bool,
     possibleLabels: PropTypes.object,
     selectedLabels: PropTypes.array,
-    contentClass: PropTypes.string,
+    contentClass: PropTypes.string
```

Comment:
I like to use dangling commas for the reasons outlined [here](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8). I also believe it's part of the Airbnb spec.

Reply:
Agreed. My editor should be adding these not removing them. I'll look into why it's working backwards.

Reply:
I’ll fix these after #159 is in dev, so I can do it systematically.